packages/CodingStandard/tests/Fixer/LineLength/BreakMethodCallsFixer/correct/correct2.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+class SomeClass
+{
+    public function someFunction()
+    {
+        $this->assertSame([
+            'one' => 1,
+            'two' => 2,
+        ], $parameterProvider->provide());
```

Comment:
I'd like to see how the fixer would format this if it was on one line. Can you add a test like that?

off topic: Personally I'd like to have a fixer that would change even the code above to this:

```php
$this->assertSame(
    [
        'one' => 1,
        'two' => 2,
    ],
    $parameterProvider->provide()
);
```

Reply:
> I'd like to see how the fixer would format this if it was on one line. Can you add a test like that?

It would not touch it, since its too short. Could you provide code that should be touched and you have in mind?

> off topic

I don't know how I'd enforce this, but if you'd need help to build it for your own needs, here I am.

Reply:
@enumag Gimme a sec, I got the use case

Reply:
@enumag Like this one? https://github.com/Symplify/Symplify/pull/585/commits/c7f9490fd5f1d7361c6a762e768c1299d42ab650